test/pretest.js

```diff
@@ -0,0 +1,12 @@
+var fs = require('fs');
+var path = require('path');
+var joinPath = path.join.bind(path, __dirname);
+
+fs.readdirSync(joinPath('ripple-lib')).forEach(function(fileName) {
```

Comment:
I'm not super fond of this, but I imagine these tests are pretty much at the EOL at this point in time.